src/hooks/use-run-tsx/index.tsx

```diff
@@ -147,11 +153,11 @@ export const useRunTsx = ({
 
         const primaryKey = componentExportKeys[0]
 
-        const UserElm = (props: any) =>
+        const userElm = (props: any) =>
```

Comment:
we should revert this by React convention- components should be PascalCase

```suggestion
        const UserElm = (props: any) =>
```

Reply:
I think these changes needs to be added to biome, else on save we will get this

Reply:
just `// @biome-ignore`, but idk how this file passed in the past if biome doesn't accept it?

Reply:
no worries!